package.json

```diff
@@ -36,6 +36,9 @@
     "test:p2p": "mocha --timeout 10000 -r ts-node/register test/p2p/*",
     "test:p2p:watch": "mocha --timeout 10000 -r ts-node/register test/p2p/*  --watch --watch-extensions ts",
     "test:crypto": "mocha --timeout 10000 -r ts-node/register test/crypto/*",
+    "test:sim:build": "(npm run compile:dev && mkdir -p test/simulation/temp/xud && cp -a dist test/simulation/temp/xud && cp -a bin test/simulation/temp/xud && cp package.json test/simulation/temp/xud && cd test/simulation && ./docker-build.sh)",
+    "test:sim:clean": "(cd test/simulation && ./docker-clean.sh)",
```

Comment:
Is there a possibility that the containers stay up once the test suite has failed/passed? Maybe it's worth considering running the clean script before running `test:sim:run`?

Reply:
Do you mean just for the travis config? Or here in the script.

Reply:
In the script. My use case is:
1. I run the simtests and the containers stay up or have a pending simtest suite running
2. I start `test:sim:run` again

Would this scenario cause issues?

Reply:
Containers don’t supposed to stay up once failed/passed (if it happens, I will fix it).
If you run `test:sim:run` while already having another instance running, it will just create another test container - I don’t think it’s a problem.